src/PortfolioGenerator.java

```diff
@@ -1,5 +1,48 @@
+import java.util.Scanner;
+
 public class PortfolioGenerator {
     public static void main(String[] args) {
-        // Write your code here ... ⬇
+        
+	String name;
+	String emailAddress;
+	int progLanguages;
+	String personalInfo;
+	int achievementShare;
+	
+	Scanner in = new Scanner(System.in);
+
+	System.out.print("Please enter your name: ");
+	name = in.nextLine():
+
+	System.out.print("Please enter your Email address: ");
+	emailAddress = in.nextline();
```

Comment:
This needs to be fixed
```suggestion
	emailAddress = in.nextLine();
```